test/bdd/features/malformed_url.feature

```diff
@@ -4,9 +4,9 @@ Feature: Check if there is an existing URL is malformed
   So that I can view the results
 
 Scenario: Checking if it's there my URL is malformed
-  Given I have a malformed URL of "htt://www.nicelife"
```

Comment:
This test shouldn't need to change. Can you replace this file with the one from 2.2.0 (https://github.com/ukwa/w3act/blob/2.2.0/test/bdd/features/malformed_url.feature) and verify it is all okay?